docs/piwiks-ini-configuration.md

```diff
@@ -1,7 +1,5 @@
 ---
-category: Develop
-previous: piwik-configuration
-next: plugin-settings
+category: CoreDevelop
```

Comment:
Moved to core development as only interesting for core developers. We need a new guide for config see #67